include_core/OMR/TypeTraits.hpp

```diff
@@ -98,6 +98,17 @@ struct RemoveCvRef : RemoveCv<typename RemoveReference<T>::Type> {};
 // template <typename T>
 // struct RemoveCvRef<T&&> : RemoveCvRef<T> {};
 
+/// Remove one layer of non-cv-qualified pointers
+template <typename T>
+struct RemoveNonCvPointer : TypeAlias<T> {};
```

Comment:
I use a different naming convention here than is used for `IsPointer` (i.e. `RemoveNonCvPointer` instead of `RemovePointerBase`). My thinking is that names are clearer if they describe what a type trait does rather than how it's used. What do others think?

Reply:
Yeah, that makes sense to me. I guess we should update the others to match? Do you want to do it or should I?